packages/app/src/cli/api/graphql/app-management/queries/apps.graphql

```diff
@@ -1,15 +1,22 @@
-query listApps {
-  apps {
-    id
-    key
-    activeRelease {
-      id
-      version {
-        name
-        appModules {
-          ...ReleasedAppModule
+query listApps($query: String) {
+  appsConnection(query: $query, first: 50) {
```

Comment:
Check -- we're going from all (presumably -- ignore if not) to 50. How does that match up to partners? Do we have sane ordering for these (e.g. most recent first)?

Reply:
1. `first` is required, so we need to pass some kind of value here.
2. 50 seems reasonable, I don't think we should have users scrolling manually beyond that.
3. I thought I recalled Partners using 50 as the max, but I can't seem to find it at the moment... That said, client-side we limit it to 25 [here](https://github.com/Shopify/cli/blob/main/packages/app/src/cli/api/graphql/find_org.ts#L9). So it's the same order of magnitude. Maybe worth checking, but it's easily changed and not a reason to hold up the PR at the moment IMO.
4. Order is the default dictated by the server, currently the most recently updated. It will match what they see in the dashboard; it feels like consistency is good.